float64 linearX
float64 linearY
float64 linearZ
float64 angularX
float64 angularY
float64 angularZ
float64 velLinearX
float64 velLinearY
float64 velLinearZ
float64 velAngularX
float64 velAngularY
float64 velAngularZ